modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -6,10 +6,21 @@
 
 include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
+[NOTE]
+{productname} 7.0 is _licensed under GPL Version 2 or later_. This version introduces a new `license_key`  configuration setting that gives self-hosted users the ability to select a usage under the GPL or to authenticate their paid license with Tiny.
+
 [[overview]]
 == Overview
 
-{productname} 7.0 was released for {enterpriseversion} and {cloudname} on Wednesday, March 20^th^, 2024. These release notes provide an overview of the changes for {productname} 7.0, including:
+{productname} 7.0 was released for {enterpriseversion} and {cloudname} on Wednesday, March 20^th^, 2024. 
+
+The release brings new features essential for content producers and managers in the modern workplace. These features improve efficiency through simplified sharing of content and providing improved tools for the asynchronous collaboration needs of remote teams.
+
+* **Revision History**: View changes from one content version to another
+* **Document Converters**: Import content from MS Word. Export content to MS Word or PDF documents
```

Comment:
```suggestion
* **Document Converters**: Import content from Microsoft Word. Export content to Microsoft Word or PDF documents.
```

Reply:
https://github.com/tinymce/tinymce-docs/blob/staging/docs-7/CONTRIBUTING.md#use-proper-names-for-programs-and-languages